.env.example

```diff
@@ -1 +1,3 @@
 SENTRY_AUTH_TOKEN=
+EXPO_PUBLIC_LOG_LEVEL=debug
```

Comment:
This env var will need to be added to prod builds, probably set to `info` but up for discussion